teams/teams-ps/teams/Test-CsTeamsShiftsConnectionValidate.md

```diff
@@ -32,7 +32,17 @@ This cmdlet validates Workforce management (WFM) connection settings. It validat
 PS C:\> $InstanceName = "test instance name"
 PS C:\> $WfmUserName = "WfmUserName"
 PS C:\> $plainPwd = "plainPwd"
-PS C:\> Test-CsTeamsShiftsConnectionValidate -ConnectorId "6A51B888-FF44-4FEA-82E1-839401E9CD74" -ConnectorSpecificSettingAdminApiUrl "https://nehstdevwfm02.contoso.com/retail/data/wfmadmin/api/v1-beta2" -ConnectorSpecificSettingCookieAuthUrl "https://nehstdevwfm02.contoso.com/retail/data/login" -ConnectorSpecificSettingEssApiUrl "https://nehstdevfas01.contoso.com/retail/data/wfmess/api/v1-beta1" -ConnectorSpecificSettingFederatedAuthUrl "https://nehstdevfas01.contoso.com/retail/data/login" -ConnectorSpecificSettingLoginPwd $plainPwd -ConnectorSpecificSettingLoginUserName $WfmUserName -ConnectorSpecificSettingRetailWebApiUrl "https://nehstdevwfm02.contoso.com/retail/data/retailwebapi/api/v1" -ConnectorSpecificSettingSiteManagerUrl "https://nehstdevfas01.contoso.com/retail/data/wfmsm/api/v1-beta2" -Name $InstanceName
+PS C:\> Test-CsTeamsShiftsConnectionValidate -ConnectorId "6A51B888-FF44-4FEA-82E1-839401E00000" -ConnectorSpecificSettings (New-Object Microsoft.Teams.ConfigAPI.Cmdlets.Generated.Models.ConnectorSpecificBlueYonderSettingsRequest -Property @{ AdminApiUrl = "https://contoso.com/retail/data/wfmadmin/api/v1-beta3"; SiteManagerUrl = "https://contoso.com/retail/data/wfmsm/api/v1-beta4"; EssApiUrl = "https://contoso.com/retail/data/wfmess/api/v1-beta2"; RetailWebApiUrl = "https://contoso.com/retail/data/retailwebapi/api/v1"; CookieAuthUrl = "https://contoso.com/retail/data/login"; FederatedAuthUrl = "https://contoso.com/retail/data/login"; LoginUserName = "PlaceholderForUsername"; LoginPwd = "PlaceholderForPassword" }) -Name $InstanceName
```

Comment:
Is there more documentation about the AdminApiUrl/SiteManagerApiUrl/EssApiUrl and where one would find that? I see they are marked as betas in the examples. Also, while I see a CID/Secret, what are the LoginUserName/LoginPwd used for and will this cmdlet run with MFA enabled?

Reply:
All those values depend on customer managed third party systems so the actual values depend on their own installations.
Docs are here, still under review atm:
https://review.docs.microsoft.com/en-us/microsoft-365/frontline/shifts-connectors?view=o365-worldwide&branch=v-lanachin-ukg